modules/gitops-release-notes-1-18-0.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes-1-18-0.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-18-0_{context}"]
+= Release notes for {gitops-title} 1.18.0
+
+{gitops-title} 1.18.0 is available on {OCP} 4.12, 4.14, 4.15, 4.16, 4.17, 4.18, and 4.19.
+
+[id="errata-updates-1-18-0_{context}"]
+== Errata updates
+
+RHEA-XXXX:NNNN - {gitops-title} 1.18.0 security update advisory::
+Issued: 2025-09-24
++
+The list of enhancements that are included in this release are documented in the following advisory:
++
+* link:https://access.redhat.com/errata/RHEA-XXXX:NNNN[RHEA-XXXX:NNNN]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="new-features-1-18-0_{context}"]
+== New features
+
+[id="fixed-issues-1-18-0_{context}"]
+== Fixed issues
+
+[id="deprecated-features-1-18-0_{context}"]
+== Deprecated and removed features
+
+Removed support for Keycloak in {gitops-title}::
+In {gitops-title} 1.18, the support for Keycloak-based authentication is removed. As an alternative, you can migrate to Dex or configure a self-managed Red Hat Build of Keycloak (RHBK) instance. Dex is the default authentication provider in {gitops-title}. It integrates with the OpenShift OAuth server, enabling users to log in to Argo CD with their existing OpenShift users and groups. For more information, see "Configuring SSO for Argo CD using Dex". If you want to use Keycloak, you must deploy and manage the link:https://docs.redhat.com/en/documentation/red_hat_build_of_keycloak/26.0/html/operator_guide/installation-[Red Hat Build of Keycloak (RHBK)] instance and manually configure the integration in the Argo CD custom resource.
```

Comment:
Can we change “If you want to use Keycloak” to “If you still want to use Keycloak,” since Keycloak is deprecated? This way, we can make it clear to users that Keycloak is deprecated but still an option. What do you think?

Reply:
Fixed.